libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteSourceProvider.java

```diff
@@ -4,7 +4,7 @@
 import com.mapbox.mapboxsdk.style.sources.GeoJsonOptions;
 import com.mapbox.mapboxsdk.style.sources.GeoJsonSource;
 
-class MapRouteSourceProvider {
+public class MapRouteSourceProvider {
```

Comment:
Does this class have to be public?

Reply:
I can't access it from the MapRouteLine class if it's not public. The compiler complains "public function exposes its public/package/ parameter type...

Reply:
Same as above re: `internal` package